harness/src/main/java/org/jboss/pvt/harness/configuration/PVTConfiguration.java

```diff
@@ -8,10 +8,29 @@
  */
 public interface PVTConfiguration
 {
+    public static final String PRODUCT_VERSION_KEY = "PRODUCT_VERSION";
+    public static final String PRODUCT_VERSION_LAST_KEY = "PRODUCT_VERSION_LAST";
```

Comment:
These ones merge from original GenericTestSuite, someones are needed for TestConfig, will update if the api works later.